modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotEventsPublisherDaemon.scala

```diff
@@ -4,25 +4,29 @@ import cats.effect.Async
 import cats.effect.std.Queue
 import cats.syntax.all._
 
+import org.tessellation.dag.domain.block.L1Output
 import org.tessellation.domain.aci.StateChannelOutput
 import org.tessellation.sdk.domain.Daemon
 import org.tessellation.sdk.domain.gossip.Gossip
 import org.tessellation.sdk.infrastructure.consensus.message.ConsensusEvent
+import org.tessellation.security.signature.Signed
 
 import fs2._
 
 object GlobalSnapshotEventsPublisherDaemon {
 
   def make[F[_]: Async](
     stateChannelOutputs: Queue[F, StateChannelOutput],
-    dagEvents: Stream[F, DAGEvent],
+    l1OutputQueue: Queue[F, Signed[L1Output]],
     gossip: Gossip[F]
   ): Daemon[F] = Daemon.spawn {
     Stream
       .fromQueueUnterminated(stateChannelOutputs)
       .map(_.asLeft[DAGEvent])
       .merge(
-        dagEvents
+        Stream
+          .fromQueueUnterminated(l1OutputQueue)
```

Comment:
is that queue used here, and only here? otherwise, events may be consumed by another subscriber

Reply:
It's used only here